Core/OfficeDevPnP.Core/Framework/Provisioning/ObjectHandlers/ObjectHandlerBase.cs

```diff
@@ -214,8 +214,14 @@ protected string TokenizeXml(string xml, Web web = null)
                 xml = masterPageRegex.Replace(xml, subsite ? "{sitecollection}/_catalogs/masterpage" : "{masterpagecatalog}");
 
                 // Site
-                var siteRegex = new Regex(web.ServerRelativeUrl.Replace("/", @"\/"));
-                xml = siteRegex.Replace(xml, "{site}");
+                var siteRegexReplacement = "{site}";
+                // If we are in the root site collection with just / as ServerRelativeUrl then we cannot replace all / with {site}, otherwise the urls will look like "{site}_layouts/15/images"
+                if (web.ServerRelativeUrl == "/")
+                    siteRegexReplacement += "/";
+
+                xml = Regex.Replace(xml, "(\"" + web.ServerRelativeUrl + ")(?!&)", "\"" + siteRegexReplacement, RegexOptions.IgnoreCase);
```

Comment:
these replace tokens have been copied from https://github.com/SharePoint/PnP-Sites-Core/blob/dev/Core/OfficeDevPnP.Core/Framework/Provisioning/ObjectHandlers/ObjectPageContents.cs#L291 as that handles XML too